nrf_security/src/core/CMakeLists.txt

```diff
@@ -5,9 +5,27 @@
 #
 
 if (CONFIG_PSA_CORE_BUILTIN)
-  add_subdirectory(builtin)
+  include(builtin/config.cmake)
+elseif(CONFIG_PSA_CORE_OBERON)
+  include(nrf_oberon/config.cmake)
 endif()
 
-if (CONFIG_PSA_CORE_OBERON)
-  add_subdirectory(nrf_oberon)
+if (CONFIG_PSA_CORE_BUILTIN OR CONFIG_PSA_CORE_OBERON)
```

Comment:
Related to the comment on subsequent lines: Can't this if-sentence be skipped. You are either using the builtin Mbed TLS core or Oberon PSA core

Reply:
commit no longer exists